constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java

```diff
@@ -16,13 +16,6 @@
 
 package androidx.constraintlayout.widget;
 
-import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
-import static android.view.ViewGroup.LayoutParams.WRAP_CONTENT;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.MATCH_CONSTRAINT_SPREAD;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.MATCH_CONSTRAINT_WRAP;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.PARENT_ID;
-import static androidx.constraintlayout.widget.ConstraintLayout.LayoutParams.UNSET;
```

Comment:
Seems like a bug from importing a static class before the class it depends on.
ie:
Importing these bellow
```
import android.view.View;
import android.view.ViewGroup;
```
Should work

Reply:
Alternatively removing these static imports and calling the constants by the class should also do the trick.

Ie: Change mentions of
```Java
MATCH_CONSTRAINT_SPREAD
MATCH_CONSTRAINT_WRAP
PARENT_ID
UNSET
```
To
```Java
LayoutParams.MATCH_CONSTRAINT_SPREAD
LayoutParams.MATCH_CONSTRAINT_WRAP
LayoutParams.PARENT_ID
LayoutParams.UNSET
```

Reply:
Doing a quick pass for the easy fix now. Will revisit this issue later. Thanks for the suggestions!